pkg/registry/minion/rest.go

```diff
@@ -91,6 +91,9 @@ func NodeToSelectableFields(node *api.Node) labels.Set {
 
 // MatchNode returns a generic matcher for a given label and field selector.
 func MatchNode(label labels.Selector, field fields.Selector) generic.Matcher {
+	if name, ok := field.RequiresExactMatch("name"); ok {
```

Comment:
I see, you added this in your change. Yes, just name by itself is incorrect. The field labels should match the access you would do to get the field in the object. "name" in v1beta3 is stored in .metadata.name. I will fix. This should have been caught in review, it's an api error.